db/migrate/20200303102215_change_phony_benchmark.rb

```diff
@@ -0,0 +1,9 @@
+class ChangePhonyBenchmark < ActiveRecord::Migration[5.2]
+  def up
+    ::Xccdf::Benchmark.find_by(ref_id: 'phony_ref_id').update(version: '0.0.0')
+  end
+
+  def down
+    ::Xccdf::Benchmark.find_by(ref_id: 'phony_ref_id').update(version: 'v0.0.0')
+  end
+end
```

Comment:
I'd appreciate a slightly more descriptive migration name. Also, down and up shouldn't really be symmetrical for this case (down should be no-op probably); if they should be though, we should just use `change` rather than up/down.